test_suite/testdata/go/builtins.star

```diff
@@ -33,7 +33,7 @@ assert_(4 not in (1, 2, 3))
 assert_(123 in {123: ""})
 assert_(456 not in {123:""})
 
-# _inconsistency_: valid syntax in go, java. Invalid in python, rust (list unhashable)
+# _inconsistency_: valid syntax in go. Invalid in python, rust, java (list unhashable)
```

Comment:
Nitpick: the syntax is perfectly fine. It's the dynamic behavior that varies by implementation.